src/main/java/me/karakelley/tictactoe/UI.java

```diff
@@ -3,15 +3,21 @@
 import java.io.PrintStream;
 
 public class UI {
```

Comment:
What is the responsibility of the UI class? I would expect it to only know how to display messages to and get input from user. Here it also knows about the Board class. Maybe when you move the logic from the Main class in some kind of Runner, that will be the one that coordinates what the UI displays to the user, be it a welcome message or a formatted board